daemon/cmd/policy.go

```diff
@@ -248,6 +248,9 @@ func (d *Daemon) PolicyAdd(rules policyAPI.Rules, opts *policy.AddOptions) (newR
 // was not able to be imported.
 func (d *Daemon) policyAdd(sourceRules policyAPI.Rules, opts *policy.AddOptions, resChan chan interface{}) {
 	policyAddStartTime := time.Now()
+	if !opts.ProcessingStartTime.IsZero() {
```

Comment:
Looks like this line is causing a nil ptr error in the Travis run

Reply:
Forgot to do a nil check on `opts` :facepalm:. Pushed a fix.